docs/content/Cube.js-Backend/REST-API.md

```diff
@@ -279,4 +280,82 @@ curl \
  -G \
  --data-urlencode 'queryingOptions={"timezone":"UTC"}' \
  http://localhost:4000/cubejs-api/v1/run-scheduled-refresh
-````
+```
+
+### /readyz
+
+Returns the ready state of the deployment.
```

Comment:
Should we add details that:

For single tenant: It's open a new connection for default datasource?
For multi tenancy: It checks only created connections, if no connections exist it will return OK.